src/legacy/core_plugins/interpreter/public/functions/esaggs.ts

```diff
@@ -17,25 +17,34 @@
  * under the License.
  */
 
-import { get } from 'lodash';
+import { get, has } from 'lodash';
 import { i18n } from '@kbn/i18n';
-import { CourierRequestHandlerProvider } from 'ui/vis/request_handlers/courier';
 // @ts-ignore
 import { AggConfigs } from 'ui/vis/agg_configs';
 import { createFormat } from 'ui/visualize/loader/pipeline_helpers/utilities';
 import chrome from 'ui/chrome';
 
 // need to get rid of angular from these
 // @ts-ignore
-import { SearchSourceProvider } from 'ui/courier/search_source';
-import { FilterBarQueryFilterProvider } from 'ui/filter_manager/query_filter';
-import { IndexPatternsProvider } from '../../../data/public';
-
-const courierRequestHandlerProvider = CourierRequestHandlerProvider;
-const courierRequestHandler = courierRequestHandlerProvider().handler;
+import { TimeRange } from 'src/plugins/data/public';
+import { SearchSource } from '../../../../ui/public/courier/search_source';
```

Comment:
Two imports from same file

Reply:
one is type and the other is js class ... for some reason putting them in one line didn't work correctly.